### DISCUSSION TITLE
Menue Optimisation > 1.15

### DISCUSSION THREAD

**Discussion by ludwich66** [webspiderteam/Bluetooth-DMM-For-Windows#16] [Ideas]
Here my draft to optimize Menu 

Better Structure, do Things in correct Order step by Step
All relevant Infos in Mainmenü
Add Attributs to store Legal and License Info present

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/12202733/182698110-6fd5366c-24db-46a3-9797-0dff44a710ac.png">

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/12202733/182698174-c8100f58-2485-48ff-ade1-a81cc42b3966.png">

Lets talk

ludwich

**Comment by webspiderteam** [comment]
Why do you think language selection needs a menu item? İn my mind you need it once as most of settings also language is mostly detecting automatic takes info from your system if it has language same with your device

**Reply by ludwich66** [reply]
in my opinion its a logical way best help for users

follow the user experiance - made a user-story this kind:

Found App 
Install App

1 Start App
2 check or change language (yes you use automatic setting :-))
3 select DMM/DMM's
4 choose kind of display mode
5 Select type of Export or Datatranfer

1-6 Use App-Help / Github-Project  / Github-Wiki / Github-discussions if anything in the flow is not working or is not logical

All just suggestions
;-) 

and by for today!

**Reply by webspiderteam** [reply]
Its the firts time you opened the app :) After you set it you will do

1. Start App
2. Connect Device
3. Set View Mode (if you didn't set it automatic in settings)
4. Use app
5. If you need Export data

am I right?

**Comment by ludwich66** [comment]
Information to my used Practice:
Used technics from Agile_software_development-medods
Q: Agile_software_development
https://en.wikipedia.org/wiki/Agile_software_development

Q: UserStory
https://en.wikipedia.org/wiki/User_story

Q: User / Persona
https://en.wikipedia.org/wiki/Persona_(marketing)

**Comment by webspiderteam** [comment]
Ok. Let me show you some examples.
OBS Studio
![image](https://user-images.githubusercontent.com/85828505/182827444-5d797105-4eae-4b05-ba26-bd3a90c04cdc.png)

Acrobat Reader
![image](https://user-images.githubusercontent.com/85828505/182827590-31d9dfe4-7603-4a7c-97ab-d85b6f164833.png)

7Zip
![image](https://user-images.githubusercontent.com/85828505/182827728-9b4c1434-992a-4388-9642-af214c491d14.png)

InkScape
![image](https://user-images.githubusercontent.com/85828505/182827956-a4079a7e-eb48-4c8b-8d39-aadd6028015e.png)

Notepad++
![image](https://user-images.githubusercontent.com/85828505/182828102-a279ef66-7f1f-4e69-8743-16a95db54ed1.png)

PyCharm
![image](https://user-images.githubusercontent.com/85828505/182829777-fa83a0ba-d31f-4f3b-b762-926cbde4a0bd.png)

Language selection is one time setted thing and users mostly know that you can find it in settings also It's the way keep menu clean and simple.

**Reply by ludwich66** [reply]
Hi Engin great work,

show and compare ;-)

They all approach the topic differently, depending on the weighting.
The first step could be easy, take your gear to the left in the menu and you're good to go.
The teaching says, focus on the flow of work, but of course also, as you say, how often something is needed.
-> e.g. Favorites, desktop icons, last used.
I personally like OBS Studio best. (best use of Icons / International to grasp quickly
but set language separate to the left menu (adobe, 7Zip)

I like that we can discuss here! +++
There is no right or wrong either!
... and you do the work yourself :-)
L

**Comment by webspiderteam** [comment]
I will put the settings in list and also I will makke a sub menu for views thats better I think. But I dont know how to make help page:)

**Reply by ludwich66** [reply]
--> Help page: set first a link to Github-WiKi
other entries on demand, see later

If the app interface is ok, I will "paint" again.

Do you already have a new build with fixed MQTT export ? I could then hunt bugs on the side

**Reply by webspiderteam** [reply]
Yes ı did fixed. Publishing in a minute

**Comment by ludwich66** [comment]
If you use new Icons please add this info under "about ?)"

Ideas for Icons are inspirated or used Partly by Icons  from The Noun Project
https://thenounproject.com

visible by "Kevin"
about by "The Icon Z"
bluetooth-connected by "Zeeshan Afzal"
chart by "barracuda"
ever by "WEBTECHOPS LLP"
gauge by "Arthur Shlain"
hamburger by "Lightspeed Flame"
language by "Aneeque Ahmed"
setting by "Lina Rafia"
setting by "The Deserve"
top by "Muneer A. Safiah"

**Reply by webspiderteam** [reply]
> Ideas for Icons are inspirated or used Partly by Icons from The Noun Project https://thenounproject.com
> 
> visible by "Kevin" about by "The Icon Z" bluetooth-connected by "Zeeshan Afzal" chart by "barracuda" ever by WEBTECHOPS LLP gauge by Arthur Shlain hamburger by Lightspee Flame language by Aneeque Ahmed setting by Lina rafia setting by The Deserve top by Muneer A.Safiah

All of them?

**Reply by ludwich66** [reply]
Yes license said "attributation"  
info with Iconname and makername 
+ Projectpage
I modified the Icons but it is fair to appreciate the work or ideas of the makers

**Comment by ludwich66** [comment]
did you use original android App?
this is a good example of stupid service.
if it is in chinese language, it is very difficult to find the language switch to change to other/english.
In any case, you did better!

**Comment by ludwich66** [comment]
Possible Icon for Attribution
i can made it with edges 
![noun-brainstorm-2909385](https://user-images.githubusercontent.com/12202733/182864212-6efe93f8-dd34-4479-9838-8590b83a4db8.svg)
